comps/llms/text-generation/tgi/llm.py

```diff
@@ -22,16 +22,27 @@
     register_statistics,
     statistics_dict,
 )
+from comps.cores.mega.utils import ConfigError, load_model_configs
 from comps.cores.proto.api_protocol import ChatCompletionRequest
 
 logger = CustomLogger("llm_tgi")
 logflag = os.getenv("LOGFLAG", False)
 
-llm_endpoint = os.getenv("TGI_LLM_ENDPOINT", "http://localhost:8080")
-llm = AsyncInferenceClient(
-    model=llm_endpoint,
-    timeout=600,
-)
+# Environment variables
+MODEL_CONFIGS = os.getenv("MODEL_CONFIGS")
+DEFAULT_ENDPOINT = os.getenv("TGI_LLM_ENDPOINT", "http://localhost:8080")
+
+# Extract the model endpoint
+llm_endpoint = "http://localhost:8080"
+configs_map = {}
```

Comment:
To keep the code clean, the codes that get 'llm_endpoint' here should be wrapped into a funtion, and called when initializing llm object.

Reply:
Have created function to get llm_endpoint and it is being called in llm_generate function. I have kept the load_model_configs outside itself because its a static content and one time load during bringing up the service.